.github/workflows/tests.yml

```diff
@@ -13,6 +13,10 @@ env:
   HOMEBREW_CHANGE_ARCH_TO_ARM: 1
   HOMEBREW_RELOCATE_RPATHS: 1
 
+concurrency:
```

Comment:
@dawidd6 Any idea if we can do this for e.g. only macOS workers on Homebrew/brew somehow (just because they get way more congested)?

Reply:
It can be set at the job level, so I guess there would be no problem with that (looking at workflows in this repo).